tests/test_base.py

```diff
@@ -4,15 +4,41 @@
 import configparser
 import os
 import shutil
+import subprocess
 import tempfile
 from contextlib import contextmanager
+from pathlib import Path
 from typing import Iterator, Optional
 from unittest import TestCase
 
 
 class TestBase(TestCase):
     """A base class with useful utils for tests."""
 
+    def setUp(self) -> None:
```

Comment:
I tried using `setUpClass` since we only need to do this once-per-class, but for some reason it resulted in a bunch of MyPy issues. MyPy didn't allow us to even do `cls.foo = "example"`..complaining that `foo` is not defined on `Type[TestBase]`.

Because CI is so short (10 minutes), and this setup is quick to execute (<5 seconds), I figured it was better to use `setUp` than have 10 type ignore comments.

Reply:
And we make few changes in this repo anyway, so there aren't many CI runs.